policy/DecamMapper.paf

```diff
@@ -96,37 +96,17 @@ exposures: {
         level:       "None"
         tables:      raw
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
I don't think there's any need to override `level` for this dataset.  I don't think it actually does anything in this context, and certainly if some other value of `level` works for other `obs_` packages that same value will work for this one.